index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Front-End School</title>
+  <link rel="stylesheet" href="./src/styles.css">
+</head>
+
+<body>
+  <!-- A DIV which contains header & headline background image -->
+  <div class="headline-bg">
+    <header>
+      <div>
+        <ul class="top-header-list">
+          <li><img src="./Images/facebook.png" alt="facebook icon"></li>
+          <li><img src="./Images/twitter.png" alt="twitter icon"></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li class="last"><a href="#">Log out</a></li>
+        </ul>
+      </div>
+
+      <nav>
+        <!-- Desktop Navbar -->
+        <div class="desktop-nav">
+          <img class="page-logo" src="./Images/logo.png" alt="page logo">
+          <ul class="desktop-nav-list">
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#services">Our Services</a></li>
+            <li><a href="#team-container">Our Team</a></li>
+            <li><a href="#supporters">Sponsors</a></li>
+            <li class="last"><a class="list-btn" href="#">Our Projects</a></li>
+          </ul>
+        </div>
+
+        <!-- Mobile Navbar -->
+        <div>
+          <img id="hamburger-icon" class="union-icon" src="./Images/hamburger.png" alt="hamburger icon">
+
+          <!-- close icon-->
+          <a href="#" class="close-icon hidden">
+            <img src="./Images/close.png" alt="close icon">
+          </a>
```

Comment:
Your mobile menu looks amazing 👌but you can still improve it, here is how.
1. The `hamburger-icon` is white, and the original design suggested to use the dark color for your mobile menu icon, because on the white background it's hard to see the white icon. Kindly make sure to update this.

Original:
![image](https://user-images.githubusercontent.com/34779928/205142205-2156de6b-9e9b-4e4e-8708-40e5e2e30bea.png)

Yours:
![image](https://user-images.githubusercontent.com/34779928/205142273-2a1cc02f-a381-4c06-a613-f722d1cedd2c.png)